packages/palette/src/elements/Carousel/Carousel.tsx

```diff
@@ -0,0 +1,174 @@
+import React, {
+  Children,
+  createRef,
+  isValidElement,
+  useEffect,
+  useMemo,
+  useRef,
+  useState,
+} from "react"
+import styled from "styled-components"
+import { useCursor } from "use-cursor"
```

Comment:
**lulz**: Promise you won't unpublish this? 😛

Reply:
🙈

Reply:
At the very least I can't unpublish it :) — the whole problem was using Github links directly in packages/Gemfiles.